packages/i18n/src/locales/run.ts

```diff
@@ -0,0 +1,14 @@
+import { promises as fs } from 'fs'
+import { possessionOutlineTranslationEN, possessionOutlineTranslationJP } from './PossessionOutline'
+
+const basePath = `${__dirname}/../../../web/public/locales`
```

Comment:
I think it's better to use the path library to make it work safely on windows.

```suggestion
const basePath = path.join(__dirname, '../../..', 'web/public/locales')
```